linkerd/app/gateway/src/gateway.rs

```diff
@@ -98,6 +98,7 @@ where
                 profile,
                 protocol: http.version,
                 logical_addr,
+                orig_dst: todo!("eliza: fix this"),
```

Comment:
i punted on fixing the gateway stuff for now, since this is just a rough sketch.